packages/rich-text-editor/test/visual/material/rich-text-editor.test.js

```diff
@@ -66,4 +67,23 @@ describe('rich-text-editor', () => {
     ]`;
     await visualDiff(div, 'max-height');
   });
+
+  it('dark theme', async () => {
+    document.documentElement.setAttribute('theme', 'dark');
+
+    const style = document.createElement('style');
+    style.innerHTML = `${colorDark.toString().replace(':host', 'html')}`;
+    document.head.appendChild(style);
```

Comment:
Could use a fixture to have the styles removed after the test

```suggestion
    fixtureSync(`<style>${colorDark.toString().replace(':host', 'html')}</style>`);
```